.travis_scripts/create_feedstocks

```diff
@@ -17,8 +17,10 @@ export PYTHONUNBUFFERED=1
 # Install Miniconda.
 echo ""
 echo "Installing a fresh version of Miniconda."
-curl -L https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh > ~/miniconda.sh
+curl -L https://repo.continuum.io/miniconda/Miniconda3-4.6.14-Linux-x86_64.sh > ~/miniconda.sh
 bash ~/miniconda.sh -b -p ~/miniconda
+touch ~/miniconda/conda-meta/pinned
+cat "conda 4.6.14" >> ~/miniconda/conda-meta/pinned
```

Comment:
Should have been `echo`. Sorry about that. Pushed fix in commit ( https://github.com/conda-forge/staged-recipes/commit/36df43955de95d9c8785516c4afa3c68db376b9c ).